textproc/tesseract/Portfile

```diff
@@ -26,18 +24,23 @@ long_description    The Tesseract OCR engine was one of the top  3  engines in \
 
 github.master_sites ${github.homepage}/archive/${git.branch}
 
-checksums           rmd160  11fae540fdd0ec4f6f9388fae4bbde790b17ee4d \
-                    sha256  05898f93c5d057fada49b9a116fc86ad9310ff1726a0f499c3e5211b3af47ec1
+checksums           rmd160  c31e95c288d9ecfa893bebe467dc5cc5f0edce5a \
+                    sha256  494d64ffa7069498a97b909a0e65a35a213989e0184f1ea15332933a90d43445 \
+                    size    3571750
 
 if {${name} eq ${subport}} {
-    depends_build-append    port:pkgconfig
+    depends_build-append    port:pkgconfig \
+                            port:autoconf \
+                            port:automake \
+                            port:libtool \
+                            port:asciidoc
 
     depends_lib             port:zlib \
                             port:libpng \
                             port:leptonica \
                             port:jpeg
 
-    cmake.out_of_source     yes
+    configure.cmd           ./autogen.sh && ./configure
```

Comment:
`./autogen.sh` should be set in `autoreconf.cmd`. e.g.

```
use_autoreconf      yes
autoreconf.cmd      ./autogen.sh
autoreconf.args
```

Reply:
Thanks. Done.